tasks/graylog_install.yml

```diff
@@ -46,6 +55,37 @@
     with_items:
       - httpd_can_network_connect
       - httpd_can_network_connect_db
+    when: ansible_selinux == 'enabled'
 
   when: ansible_os_family == 'RedHat'
-  
\ No newline at end of file
+
+- block:
+  - name: "Generate systemd file"
+    template:
+      src: graylog.service.j2
+      dest: /usr/lib/systemd/system/graylog-server.service
+    notify: Restart Graylog
+
+  - name: Ensure service is enabled
+    service:
+      name: "{{ graylog_service_name }}"
+      enabled: yes
+      state: started
+
+  - name: Ensure directory exist
+    file:
+      path: "{{ item }}"
+      state: directory
+      owner: root 
+      group: root
```

Comment:
Do we use `root` or `graylog`?
In systemd file the user running Graylog is `graylog`, will they have access to required path?